lib/actions/deploy.coffee

```diff
@@ -58,6 +63,15 @@ performUpload = (image, token, username, url, size, appName) ->
 
 	uploadToPromise(post, size)
 
+uploadLogs = (logs, token, url, buildId, username, appName) ->
+	request = require('request')
+	url = url || process.env.RESINRC_RESIN_URL
+	request.post
+		url: getBuilderLogPushEndpoint(url, buildId, username, appName)
+		auth:
+			bearer: token
+		body: Buffer.from(logs).toString('base64')
```

Comment:
From https://github.com/request/request
> body - entity body for PATCH, POST and PUT requests. Must be a Buffer, String or ReadStream. If json is true, then body must be a JSON-serializable object.

That makes me think you can just pass the `Buffer` object directly